### PR TITLE
Make project form on create and edit to look more similar

### PR DESCRIPTION
**Pull request by suricactus**
Also renamed the elements to have the same name to reduce cognitive needs.

| before | after |
|-|-|
| image | image |

### CODE REVIEW

File: qfieldsync/gui/cloud_create_project_widget.py [hunk 1]
```diff
@@ -99,18 +99,27 @@ def __init__(
         self.nextButton.clicked.connect(self.on_next_button_clicked)
         self.backButton.clicked.connect(self.on_back_button_clicked)
         self.createButton.clicked.connect(self.on_create_button_clicked)
-        self.dirnameButton.clicked.connect(self.on_dirname_button_clicked)
-        self.dirnameLineEdit.textChanged.connect(self.on_dirname_line_edit_text_changed)
+        self.localDirButton.clicked.connect(lambda: self.on_local_dir_button_clicked())
```
Comment (faebebin) [inline qfieldsync/gui/cloud_create_project_widget.py:102]:
```suggestion
        self.localDirButton.clicked.connect(self.on_local_dir_button_clicked)
```

File: qfieldsync/gui/cloud_create_project_widget.py [hunk 2]
```diff
@@ -99,18 +99,27 @@ def __init__(
...
-        self.dirnameButton.setMenu(QMenu())
-        self.dirnameButton.setPopupMode(QToolButton.MenuButtonPopup)
-        self.dirnameButton.menu().addAction(self.use_current_project_directory_action)
+        self.localDirButton.setMenu(QMenu())
+        self.localDirButton.setPopupMode(QToolButton.MenuButtonPopup)
+        self.localDirButton.menu().addAction(self.use_current_project_directory_action)
+
+        self.localDirOpenButton.clicked.connect(
+            lambda: self.on_local_dir_open_button_clicked()
+        )
```
Comment (faebebin) [inline qfieldsync/gui/cloud_create_project_widget.py:119]:
```suggestion
        self.localDirOpenButton.clicked.connect(
            self.on_local_dir_open_button_clicked
        )
```

**Review by faebebin — COMMENTED** [review]
Code looks good

Except, I guess the lambda wrapped callbacks are not necessary without arguments ?

File: qfieldsync/gui/cloud_create_project_widget.py (hunk 1 above)
Comment (suricactus) [reply qfieldsync/gui/cloud_create_project_widget.py]: Done!

File: qfieldsync/gui/cloud_create_project_widget.py (hunk 2 above)
Comment (suricactus) [reply qfieldsync/gui/cloud_create_project_widget.py]: Done!

**Pull request merged**